docs/develop.rst

```diff
@@ -109,21 +109,20 @@ The method defines the follwing things:
 - ``plonecli_alias``: a short name alias which will be used by plonecli
 - ``depend_on``: an optional global parent template
 
-We use here global unique template names which have the ``plone_`` prefix. Thats because other bobtemplate packages might register templates two and we want avoit naming clashes.
-
+We use here global unique template names which have the ``plone_`` prefix. Thats because other bobtemplate packages might register templates too and we want avoit naming clashes.
 
 Testing
 =======
 
 All templates and sub-templates should have tests for the structure they provide.
 
-These tests will give the developers a good starting point. So that they only need to write test for there own code.
-Also these tests will be called by Tox and on Travis to make sure all by bobtemplates.plone created structures are working and tested.
+These tests will give developers a good starting point to write tests for their own code.
+Also these tests will be called by Tox and on Travis to make sure that all the structures created by ``bobtemplates.plone`` are working and tested.
 
-To run these test we run all templates in every combination and run the tests inside the created packages.
+To run these tests we run tests for both all the templates with every combination and inside the created packages
 
-This could be for example the ``addon`` alone. Or for a package with Dexterity content types,
-first the ``addon`` template and then inside the created package the ``content_type`` sub-template.
+For example tests could be run only on ``addon``.
+Alternately, for a package with Dexterity content types, tests could be run first for the ``addon`` template, then inside the package created by the ``content_type`` sub-template.
```

Comment:
I omitted our [style guide preferred spelling of "add-on"](https://docs.plone.org/about/contributing/word_choice.html#preferred-word-choice). Please fix the above instances, and wherever else it appears in this page. Thank you!